code/modules/mob/living/living.dm

```diff
@@ -283,6 +283,34 @@
 		if(H.species.tox_mod)
 			mult = H.species.tox_mod
 
+	if(reagents.has_reagent(PRO_TOXIN))
```

Comment:
Thanks, I hate it. 

As I see it it's automatically self-defeating, though. Or convoluted and buggy.
You get pro-tox, then some toxin in your body, the pro-tox multiplies the reagent you have (like Plasma) in  `pro_toxin/on_mob_life`, but then removes the exact same amount in `/mob/living/proc/adjustToxLoss`.

And since it has `adjustToxLoss(-1.5)` it looks on the outset that protoxin will make these sources heal you, but you invert the multiplier, so what looks like a heal code-wise is in fact a hurt in game. It's convoluted as hell.